scripts/xephyr

```diff
@@ -3,12 +3,13 @@
 HERE=$(dirname $(readlink -f $0))
 SCREEN_SIZE=${SCREEN_SIZE:-800x600}
 XDISPLAY=:1
+LOG_LEVEL=${LOG_LEVEL-INFO}
```

Comment:
Alright, I wasn't sure, which is why I asked, if you change that I'd be happy to pull it in!